entity-framework/core/get-started/wpf.md

```diff
@@ -0,0 +1,156 @@
+ms.author: jeliknes
+ms.date: 07/24/2020
+uid: core/get-started/wpf
+---
+# Getting Started with WPF
+
+This step-by-step walkthrough shows how to bind POCO types to WPF controls in a "main-detail" form. The application uses the Entity Framework APIs to populate objects with data from the database, track changes, and persist data to the database.
+
+The model defines two types that participate in one-to-many relationship: **Category** (principal\\main) and **Product** (dependent\\detail). The WPF data-binding framework enables navigation between related objects: selecting rows in the master view causes the detail view to update with the corresponding child data.
+
+The screen shots and code listings in this walkthrough are taken from Visual Studio 2019 16.6.5.
+
+> [!TIP]
+> You can view this article's [sample on GitHub](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/WPF).
+
+## Pre-Requisites
+
+* You need to have Visual Studio 2019 16.3 or later installed with the **.NET desktop workload** selected to complete this walkthrough.
+    
+    For more information about installing the latest version of Visual Studio, see [Install Visual Studio](/visualstudio/install/install-visual-studio).
+
+## Create the Application
+
+1. Open Visual Studio
+2. On the start window, choose **Create new project**.
+3. Search for "WPF," choose **WPF App (.NET Core)** and then choose **Next**.
+4. At the next screen, give the project a name, for example, **GetStartedWPF**, and choose **Create.**
+
+## Install the Entity Framework NuGet packages
+
+1. Right-click on the solution and choose **Manage NuGet Packages for Solution...**
+
+    ![Manage NuGet Packages](_static/wpf-tutorial-nuget.jpg)
+
+1. Type `entityframeworkcore.sqlite` in the search box.
+1. Select the **Microsoft.EntityFrameworkCore.Sqlite** package.
+1. Check the project in the right pane and click **Install**
+
+    ![Sqlite Package](_static/wpf-tutorial-sqlite.jpg)
+
+1. Repeat the steps to search for `entityframeworkcore.proxies` and install **Microsoft.EntityFrameworkCore.Proxies**.
+
+> [!NOTE]
+> When you installed the Sqlite package, it automatically pulled down the related **Microsoft.EntityFrameworkCore** base package. The **Microsoft.EntityFrameworkCore.Proxies** package provides support for "lazy-loading" data. This means when you have entities with child entities, only the parents are fetched on the initial load. The proxies detect when an attempt to access the child entities is made and automatically loads them on demand. 
+
+## Define a Model
+
+In this walkthrough you will implement a model using "code first." This means that EF Core will create the database tables and schema based on the C# classes you define.
+
+Right-click on the project in **Solution Explorer** and choose **Add &gt; Class...**. Give it the name: `Product.cs` and populate it like this:
+
+**`Product.cs`**
+
+[!code-csharp[](../../../samples/core/WPF/GetStartedWPF/GetStartedWPF/Product.cs)]
```

Comment:
```suggestion
[!code-csharp[Main](../../../samples/core/WPF/GetStartedWPF/GetStartedWPF/Product.cs)]
```

Reply:
@smitpatel what is the difference between empty and Main? is the usage documented somewhere?

Reply:
I don't know the difference. But we use it everywhere so consistency. :trollface: 

Actual guidance is bit more complicated. https://review.docs.microsoft.com/en-us/help/contribute/code-in-docs?branch=master

Reply:
Interesting, and none of those reference the style we use. I don't see any specific difference in highlighting, so unless you object I'd prefer not to name it unless there is a compelling reason to do so.

Reply:
I am fine either way. We should start using recommended guideline in future.